src/shared/components/lazyMobXTable/LazyMobXTable.tsx

```diff
@@ -787,7 +780,7 @@ export default class LazyMobXTable<T> extends React.Component<
                 // populate the cache instances with all available data for the lazy loaded columns
                 this.store.downloadDataFetcher
                     .fetchAndCacheAllLazyData()
-                    .then(allLazyData => {
+                    .then(_allLazyData => {
```

Comment:
couldn't you just replace this one with `()`?

Reply:
yes, it looks like passing in a no-arg lambda works (no type mismatch)